### ISSUE TITLE
How to add a device to a specific user?

### ISSUE THREAD

**Issue by GusevVictor** [kingmo888/rustdesk-api-server#151]
Hey, there!

Could you please explain me how to add a device to a specific user? I've setup rundesk client and can't add device to an user, because of this I can't share it.

![image](https://github.com/user-attachments/assets/8021bade-2761-43b2-9d1b-2dd28d293ada)

Best regards!

**Comment by kingmo888** [comment]
This displays all current device information, but this device information should not belong to the account you are currently logged in to. In authorized sharing, only devices that have been saved in the client logged in with one's own account should be shared. Even if you are an administrator, you should not share devices that do not belong to you.
This is my understanding.

**Comment by kingmo888** [comment]
![image](https://github.com/user-attachments/assets/88d58496-e127-4283-af6e-d00916347075)

![image](https://github.com/user-attachments/assets/b6726a4f-6265-4672-afcc-7fdcf7e3eff7)

Please look forward to the v2 (pro) version, which is more comprehensive in terms of functionality.
I have been busy with other projects recently, and this version has temporarily stopped progressing to around 90%, which will take time.

**Comment by kingmo888** [comment]
Of course, language internationalization is not included, it needs to be fully functional before adjusting to support internationalization.

**Comment by a180285** [comment]
@kingmo888 where is the v2 (pro) version?

**Comment by kingmo888** [comment]
> @kingmo888 where is the v2 (pro) version?

There is still a small task to be completed. I plan to release v2 as soon as the current project star exceeds 1k
